pages/widgets/calculator.mdx

```diff
@@ -52,6 +52,16 @@ These parameters must be supplied via `data-` attributes, otherwise the widget w
 
 ---
 
+##### `token` string
+
+We use the token as an identifier for your organisation. This way, we can:
```

Comment:
Mag ik even een taal Nazi zijn: moet het niet "We use a token as an ...."

Reply:
Sure, fixed. Heb ook nog `unique` toegevoegd.